French version of en.ftl

Natural-Brother opened the pull request:
Hello,
This is the french version of the **en.ftl** file.
except i have skipped the line 47 _"Carryalls"_ because i dont know what this is.

Natural-Brother committed:
Create fr.ftl

This is the french version of OpenRA/mods/d2k/languages/chrome
/en.ftl file.
the file is not completely translated, i have skipped line 47 "Carryalls" and line 77 to line 80.
I stopped at line 143.

Natural-Brother committed:
Update fr.ftl

This is the french version of **OpenRA/mods/d2k/languages/chrome
/en.ftl**.

I have skipped the line 47 "Carryalls" because i dont know what this is.

abcdefg30 reviewed (commented):
Hi! Please see my response at https://github.com/OpenRA/OpenRA/pull/21335#pullrequestreview-1876507472.

As for "Carryall", that refers to an aircraft that lifts and flies vehicles from one position on the map to another position on the map.

PunkPun commented:
Carryall is a combination of 2 words. Carry & all

JovialFeline commented:
For what it's worth, the original FR translation has Carryall as this:
![CarryallFRE](https://github.com/OpenRA/OpenRA/assets/4985264/cbd51619-6992-4886-9f64-07ff3bb5b848)

However, I know dreadfully little French and can't vouch for the translation's quality.

Natural-Brother committed:
Update fr.ftl

I replaced "Carryalls" by "Aile Portante" at line 47
and fixed some detail (spaces, uppercases)

Natural-Brother commented:
Even if t can't be implanted, 
can i translate the others files?

dnqbob commented:
I think you can do this later, because devs in https://github.com/OpenRA/OpenRA/issues/20693 may have plan to split "en.ftl".

Mailaender commented:
Uploaded to https://crowdin.com/editor/openra/58/en-fr

PunkPun commented:
Closing as outdated, and referencing #10475 so we don't forget
